bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
I guess another option is to pass a list of stuff things that must be passed down, as such:

```
#!/usr/bin/python-noenv PBS_DEFAULT:HOME:USER
```